src/components/Confirm/Confirm.tsx

```diff
@@ -141,18 +140,27 @@ export const Confirm = (props: ConfirmProps) => {
       if (props.crossDeviceClientError) {
         return props.crossDeviceClientError()
       }
-    } else if (status === 422) {
-      errorKey = response?.error
+      setError('' as ErrorNames)
```

Comment:
Why not `undefined` instead of empty string? The `errorProp` state , updated by `setError`, is `<ErrorProp | undefined>`

Reply:
I wanted to keep current behavior, but a more correct current behavior would be setError(undefined, undefined). 
I'll just remove the setError call as everything is undefined in that case anyway